src/main/java/apoc/periodic/PeriodicIterate.java

```diff
@@ -0,0 +1,44 @@
+package apoc.periodic;
+
+import apoc.util.Util;
+import org.neo4j.internal.helpers.collection.Pair;
+
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class PeriodicIterate {
```

Comment:
Maybe a better name could be `PeriodicUtils`?

And add a private constructor so we can prevent the initialization